src/common/pubsub.ts

```diff
@@ -475,3 +476,15 @@ export const workerSubscribe = (
     ),
   );
 };
+
+export const notifyUserPostPromoted = async (
```

Comment:
We have a newer type safe utilities for pubsub. Take a look here: https://github.com/dailydotdev/daily-api/blob/main/src/common/typedPubsub.ts and check how we use it. There are a lot of examples.

@capJavert we should consider adding a big comment that this file is deprecated